packages/core/docs/guide/cart-and-wishlist.md

```diff
@@ -26,66 +22,65 @@ export default {
       load
     };
   }
-};   
+};
 ```
 
-
 ## Adding item to the cart
 
 To add the product to the cart you can use `addItem` method:
 
 ```vue
 <template>
-  // ...
-    <ul>
-      <li
-        v-for="product in products" :key="product.id"
-      > 
-        // ...
-        <button
-          @click="addItem({ product, quantity })"
-        >
-          Add to cart
-        </button>
-      </li>
-    </ul>
-  // ...
-</template>    
-<script>     
-  import { computed } from '@vue/composition-api';
-  import { useCart } from '{INTEGRATION}';
-  export default {
-    props: {
-      products: {
-        type: Array,
-        required: true
-      }
-    },
-    setup() {
-      const {
-        addItem,
-      } = useCart();
-
-      // load cart if it wasn't loaded before
-
-      return {
-        addItem,
-      };
+  <!-- ... -->
+  <ul>
+    <li v-for="product in products" :key="product.id">
+      <!-- ... -->
+      <input type="number" v-model="quantity" />
+      <button @click="addItem({ product, quantity })">
+        Add to cart
+      </button>
+    </li>
+  </ul>
+  <!-- ... -->
+</template>
+<script>
+import { ref } from '@vue/composition-api';
+import { useCart } from '{INTEGRATION}';
+
+export default {
+  props: {
+    products: {
```

Comment:
It was proposed by Filip R. It differs because when you go to listing page it should go from somewhere else, is it right or not?

Reply:
I'm afraid I have to disagree with this. In these examples, we show *how* to do something (e.g., how to add an item to the cart), not which component should load products or how data should flow between the components.

I prefer that all examples on this page are consistent and leave the details (like the data flow between the components) to the developers. **We should show that `products` come from `useProduct` composable because this is not obvious if it comes from `props`.**

Additionally,  in all examples we name the variable `products`, but some of them are `products` from `useProduct`, others are computed properties from getters (`computed(() => cartGetters.getItems(cart.value))` or `computed(() => wishlistGetters.getItems(wishlist.value))`).
**Please use the name `products` only for products coming from `useProduct` and the names `cartItems` or `wishlistItems` for products coming from getters.**

WDYT? @AdamPawlinski @filrak @igorwojciechowski

Reply:
I removed all product props and changed the names, so please take a look if this looks fine for you @filipsobol